.eslintrc

```diff
@@ -216,6 +216,7 @@
 		"no-self-compare": "error",
 		"no-sequences": "error",
 		"no-shadow": "off",
+		"no-undef": "off",
```

Comment:
Why did you turn this off?

Reply:
I just wanted to make it explicit -- it's off by default when missing. I'd personally prefer it to be at least a warning.

Reply:
It's not off — I [added](https://github.com/zotero/translators/commit/b3c0346183e893a88351d301a4752f9f5ec66daf) `eslint:recommended`, which enables this. The client rules assumed that was set, but I forgot to copy it over.

Reply:
(This might explain some of your confusion about the client rules.)

Reply:
my bad. Removed.